Antlr.DOT/Antlr.DOT.csproj

```diff
@@ -69,6 +66,11 @@
       <StrongName>StrongName</StrongName>
     </FilesToSign>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="MicroBuild.Core">
+      <Version>0.2.0</Version>
+    </PackageReference>
+  </ItemGroup>

```

Comment:
It wasn't there before (packages.config does not affect the actual build), and it still builds successfully now, so I suppose no ;).